packages/foreman/foreman/foreman.spec

```diff
@@ -830,6 +809,15 @@ Meta package with support for plugins.
 %{_sysconfdir}/rpm/macros.%{name}-plugin
 %{_datadir}/%{name}/schema_plugin.rb
 
+%package plugin-build
+Summary: Foreman plugin RPM build support
+Group: Development/Libraries
+
+%description plugin-build
+Meta package with support for plugin RPM building.
+
+%files plugin
```

Comment:
Not only plugins need this. I'd suggest a `foreman-build` if you want to split this off from `foreman-release`.